ProjectNetwork.Services/ProjectsService.cs

```diff
@@ -39,11 +53,28 @@ public Project GetProject(int Id)
             return project;
         }
 
+//<<<<<<< HEAD
+        public List<Skill> GetMySkills(int projectId)
+        {
+            var tempProject = this._projectSkillRepository.GetAll(proj => proj.ProjectId == projectId);
```

Comment:
Why GetMySkills is part of the project service and not the skills service? And why are you getting "mySkills" from projectSkill repository and not userSkill?